### PR TITLE
consolidate all email preferences to the email settings page

### LINKED ISSUE #1000: All email settings should be on the same page
At the moment they are spread over https://www.zooniverse.org/#/settings and https://www.zooniverse.org/#/settings/email. It would be easier and more sensible to have them all on the settings/email page.

### PR DESCRIPTION
**Pull request by camallen**
Closes #1000 - I couldn't help myself when i saw it was the 1000th issue.

### CODE REVIEW

File: app/pages/settings/email.cjsx [hunk 1]
```diff
@@ -12,6 +13,28 @@ module.exports = React.createClass
   render: ->
     <div className="content-container">
       <p><strong>Project email preferences</strong></p>
```
Comment (edpaget) [inline app/pages/settings/email.cjsx]: I think this title should be able the lower group of email preferences. And this line should be "Zooniverse email preferences".

**Comment by edpaget** [comment]
Any chance you also want to add an autosave box to email preferences page for users to change their email address?

**Comment by DarrenMcRoy** [comment]
This would be really nice. We got several requests for such an option! :-)

**Comment by camallen** [comment]
@edpaget updated and rebased, should be good to go.

**Pull request merged**